### ISSUE TITLE
abnormal morphology and function of cilia

### ISSUE THREAD

**Issue by RiyaUthup** [obophenotype/human-phenotype-ontology#10836]
**Preferred term label:**abnormal morphology and function of cilia

**Synonyms**

**Definition (free text, please give PubMed ID)** 39132680

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@RiyaUthup 
can you specify the use case? Also, there are many different kinds of cilia? What diseases are you aiming to annotate?

**Comment by RiyaUthup** [comment]
The disease is hepatorenal ciliopathy.

**Comment by pnrobinson** [comment]
THanks for the suggestion. I will need to do some research, but will start with the desrcriptions here: https://pubmed.ncbi.nlm.nih.gov/39132680/
